### DISCUSSION TITLE
Github Campus expert

### DISCUSSION THREAD

**Discussion by lawalalao** [community/community#58999] [GitHub Education]
### Hi everyone,

1. Existing Technical Student Communities at EPITECH Technology Campus:
EPITECH is known for its focus on computer science and technology education. At EPITECH, you may find the following technical student communities:

- EIP (Epitech Innovative Projects): EIP is a community of students working on innovative projects, often with a technological focus. They collaborate on creating cutting-edge software applications, products, and services.

- Coding Club: EPITECH may have a coding club or programming society where students engage in coding challenges, participate in programming competitions, and exchange knowledge and skills related to software development.

- Robotics and IoT Club: EPITECH students with an interest in robotics, electronics, and the Internet of Things (IoT) may form a club to work on robotics projects, organize competitions, and explore automation technologies.

- Open-source Community: EPITECH students may contribute to open-source projects and actively participate in the open-source community. They collaborate on improving coding skills, sharing knowledge, and working on real-world projects.

2. Who is Served and Gaps at EPITECH Technology Campus:
EPITECH Technology Campus primarily serves students pursuing computer science and technology-related programs. However, there may be some gaps that can be addressed:

- Interdisciplinary collaboration: Encouraging collaboration between EPITECH students and students from other fields, such as design, business, or arts, can foster interdisciplinary projects and innovation.

- Inclusivity: Ensuring inclusivity and representation of underrepresented groups within the student tech communities is crucial. Efforts should be made to provide equal opportunities, support, and resources for all students.

3. GitHub Education Program Support at EPITECH Technology Campus:
The GitHub Campus Experts program can support the on-campus student communities at EPITECH Technology Campus. By becoming a GitHub Campus Expert, i can:

- Organize workshops and events: Conduct workshops and events that focus on leveraging GitHub for version control, collaboration, and project management. These activities can help EPITECH students enhance their coding and collaboration skills.

- Promote open-source culture: Encourage students to contribute to open-source projects by providing guidance on using GitHub for open-source development. Showcase successful open-source projects and foster a culture of collaboration and community engagement.

- Facilitate mentorship: Connect EPITECH students with experienced mentors from the industry who can guide them in their technical journey. Mentors can provide insights, advice, and support to help students grow their skills and build their professional network.

4. Supporting the Growth of Student Tech Communities at EPITECH Technology Campus:
To support the growth of student tech communities at EPITECH Technology Campus,
- Collaborate with EPITECH staff: Work closely with faculty members, EPITECH alumni, and staff to establish partnerships, leverage resources, and gain support for community-building initiatives.

- Host tech-focused events: Organize hackathons, coding competitions, and tech conferences to foster a sense of community, provide platforms for students to showcase their skills, and encourage networking.

- Establish industry connections: Forge relationships with local tech companies and invite industry professionals to campus for guest lectures, workshops, and career networking events.

- Provide project-based learning opportunities: Encourage students to work on real-world projects, either through internships, industry collaborations, or community-led initiatives. This hands-on experience can enhance their skills and expose them to practical challenges.

- Create a supportive environment: Foster a culture of inclusivity, diversity, and collaboration within the tech communities at EPITECH. Establish mentorship programs, peer support networks, and initiatives that promote equal opportunities for all students.

**Comment by samus-aran** [comment]
Hi @lawalalao ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
